src/clients/inst_scc.rb

```diff
@@ -136,8 +136,8 @@ def select_addons
       return :cancel if get_available_addons == :cancel
 
       # FIXME: workaround to reference between old way and new storage in Addon metaclass
-      @selected_addons = Registration::Addon.selected
-      ::Registration::Storage::InstallationOptions.instance.selected_addons = @selected_addons
+      # @selected_addons = Registration::Addon.selected
+      # ::Registration::Storage::InstallationOptions.instance.selected_addons = @selected_addons
```

Comment:
No dead code please, remove it completely. If needed it can be restored from the Git history.

Reply:
Ooops! This is a clear leftover. I thought that removed it before make the commit... but... 

Thanks!